### PR TITLE
Blog post detailing quick install of IPFS latest

### PR DESCRIPTION
**Pull request by RichardLitt**
This is a copy of #8 that is being pushed to a local branch so that @RichardLitt can edit it. #8 will be closed.

### CODE REVIEW

**Comment by RichardLitt** [comment]
@kyledrake @jbenet I've edited up this blogpost a bit, and added an asciinema. Thoughts?

**Comment by jbenet** [comment]
@RichardLitt possible to include an ipfs link to a rendered version to review it there?

**Comment by RichardLitt** [comment]
Sure. https://ipfs.io/ipfs/QmU88a9FPwxg5YN8BknKKc262bCqjaNvSgRFxJrt7H2vrp/2-run-ipfs-on-a-vps/

**Comment by RichardLitt** [comment]
@em-ly This is ready to go.

**Comment by em-ly** [comment]
@RichardLitt Awesome! Let's get someone to review and then we'll go live.

**Comment by VictorBjelkholm** [comment]
```
> wget https://gobuilder.me/get/github.com/ipfs/go-ipfs/cmd/ipfs/ipfs_master_linux-amd64.zip
```

URL should change to `https://dist.ipfs.io/go-ipfs/v0.4.4/go-ipfs_v0.4.4_linux-amd64.tar.gz`

File: src/2-run-ipfs-on-a-vps/index.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+```
+
+To check if your IPFS node has this content pinned, you can run `ipfs pin ls`:
+
+```
+> ipfs pin ls | grep QmT78zSuBmuS4z925WZfrqQ1qHaJ56DQaTfyMUF7F8ff5o
+QmT78zSuBmuS4z925WZfrqQ1qHaJ56DQaTfyMUF7F8ff5o recursive
+```
+
+In this example, the hash is returned because `ipfs add` pins a file passed to it by default. If the hash is not returned, then the content will be removed at the next garbage collection. To stop this from happening, let's pin it:
```
Comment (whyrusleeping) [inline src/2-run-ipfs-on-a-vps/index.md]: Should probably make it a bit more clear here that `ipfs add` does actually pin content automatically

**Comment by whyrusleeping** [comment]
mostly okay looking, needs a few updates

**Comment by RichardLitt** [comment]
@VictorBjelkholm Tried to find the link to the original discussion about the `>` prompt, and failed. Our styleguide [here](https://github.com/ipfs/community/blob/master/docs-styleguide.md) uses `>` because it is more universal.

Updated the other things; thoughts?

**Comment by VictorBjelkholm** [comment]
> @VictorBjelkholm Tried to find the link to the original discussion about the > prompt, and failed. Our styleguide here uses > because it is more universal.

Fair enough, but if we don't want to indicate the privilege needed for running the command, you need to add `sudo` in front of `cp ipfs/ipfs /usr/local/bin/` or something similar.

**Comment by RichardLitt** [comment]
@VictorBjelkholm Added a note. How does it look now?

**Comment by VictorBjelkholm** [comment]
@RichardLitt looks awesome, think this is good to go :+1:

**Comment by RichardLitt** [comment]
Sweet! Promoting this to ready.

**Pull request merged**